DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentRepository.cs

```diff
@@ -92,19 +130,59 @@ public int Update(DocumentModel entity)
         _logger.LogTrace("Document: {Document}", entity);
         using IDbConnection con = _connectionFactory.CreateConnection();
         return con.Execute(
-            "update documents set title = @Title, author = @Author, date = @Date, summary = @Summary, " +
-            "path = @Path, total_words = @TotalWords, sources_id = @SourceId " +
-            "where id = @Id",
-            new
-            {
-                entity.Title,
-                entity.Author,
-                entity.Date,
-                entity.Summary,
-                entity.Path,
-                entity.TotalWords,
-                entity.SourceId,
-                entity.Id
-            });
+                        "update documents set title = @Title, author = @Author, date = @Date, summary = @Summary, " +
+                        "path = @Path, total_words = @TotalWords, sources_id = @SourceId " +
+                        "where id = @Id",
+                        new
+                        {
+                            entity.Title,
+                            entity.Author,
+                            entity.Date,
+                            entity.Summary,
+                            entity.Path,
+                            entity.TotalWords,
+                            entity.SourceId,
+                            entity.Id
+                        });
+    }
+
+    public IEnumerable<DocumentModel> GetByAuthor(string author)
+    {
+        _logger.LogDebug("Retrieving Documents by {author} from database", author);
+        using IDbConnection con = _connectionFactory.CreateConnection();
+        return con.Query<DocumentModel>($"select * from documents where author = @Author",
```

Comment:
Fjern string interpolation så man ikke fristes til at bruge det
```suggestion
        return con.Query<DocumentModel>("select * from documents where author = @Author",
```